OpenRA.Mods.Common/Projectiles/LaserZap.cs

```diff
@@ -36,7 +36,7 @@ public class LaserZapInfo : IProjectileInfo
 
 		public readonly bool UsePlayerColor = false;
 
-		[Desc("Laser color in (A,)R,G,B.")]
+		[Desc("Laser color.")]
```

Comment:
Having the description for "Laser: Color" be "Laser color." doesn't tell us anything useful.  Just remove it.